front/tailwind.config.js

```diff
@@ -26,10 +26,6 @@ module.exports = {
           100: { DEFAULT: colors.blue[100], dark: colors.blue[900] },
           50: { DEFAULT: colors.blue[50], dark: colors.blue[950] },
         },
-        success: {
```

Comment:
should we add the keyframes here as well?

Reply:
Indeed. Correcting